src/types.rs

```diff
@@ -14,16 +14,18 @@ pub struct FunctionInfo {
     pub handler: Py<PyAny>,
     pub is_async: bool,
     pub number_of_params: u8,
+    pub validate_params: bool,
```

Comment:
Hey @kliwongan ,

Thanks for you PR 😄 . I have a few comments. 

I want to understand the reasoning behind naming the variable as `validate_params` , are we expecting a hashmap or an object in the future? If not, something like `to_validate` or `validation` would be more relevant.

Reply:
Yes, I agree. I simply wanted the user to have the choice of taking in a raw request, or define the request body using the function signature, hence the boolean variable.

Reply:
However, I wasn't sure whether or not to embed this in the FunctionInfo struct or not, as for middleware-related and websocket-related functions, validation is not really applicable?

Reply:
@kliwongan , you can make it an `Optional<bool>`

Reply:
You mean an Option<bool>? I was thinking of setting the default argument of validate as False (since dataclasses can detect those things) since an Option in Rust requires unwrapping it and checking for None, which adds an additional layer of checking, but I'm not sure how that would work with PyO3.